src/index.js

```diff
@@ -1,27 +1,110 @@
-  { description: 'complete T Do list project', completed: false, index: 2 },
-];
 const tasksList = document.querySelector('.tasks-list');
+const inputTask = document.querySelector('#input-task');
 
-const renderOldTasks = () => {
-  for (let i = 0; i < tasks.length; i += 1) {
-    const task = document.createElement('li');
-    task.textContent = `\u00a0  \u00a0 \u00a0 ${tasks[i].description}`;
-    tasksList.appendChild(task);
+class Task {
+  constructor(index, description, completed = false) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+}
+class Tasks {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  addTask(task) {
+    this.tasks.push(task);
+  }
+
+  renderTasks() {
+    this.tasks.forEach(this.renderTask);
+  }
+
+  renderTask(task) {
+    const taskLi = document.createElement('li');
+    taskLi.setAttribute('index', task.index);
+    taskLi.className = 'task-item';
+    tasksList.appendChild(taskLi);
+
+    const taskDiv = document.createElement('div');
+    taskLi.appendChild(taskDiv);
 
     const checkbox = document.createElement('input');
     checkbox.type = 'checkbox';
     checkbox.className = 'checkbox';
-    checkbox.checked = tasks[i].completed;
-    task.appendChild(checkbox);
+    taskDiv.appendChild(checkbox);
+
+    const inputText = document.createElement('input');
+    inputText.value = task.description;
+    inputText.className = 'input-text';
+    inputText.setAttribute('disabled', 'true');
+    taskDiv.appendChild(inputText);
 
-    const move = document.createElement('img');
-    move.src = 'https://drive.google.com/uc?export=download&id=1sPrm0H-RE7O346HqzOGXcqp1-lyE2xC_';
-    task.appendChild(move);
+    const editTaskImg = document.createElement('img');
+    editTaskImg.className = 'edit-task-img';
+    editTaskImg.src = 'https://drive.google.com/uc?export=download&id=1sPrm0H-RE7O346HqzOGXcqp1-lyE2xC_';
+    taskLi.appendChild(editTaskImg);
+
+    const editTaskImgTrash = document.createElement('img');
+    editTaskImgTrash.className = 'edit-task-img-trash';
+    editTaskImgTrash.style.display = 'none';
+    editTaskImgTrash.src = 'https://drive.google.com/uc?export=download&id=1FQtxnRA9DqZVonm55uyTS5Ee-CfNj7RK';
+    taskLi.appendChild(editTaskImgTrash);
+  }
+
+  deleteTask(index) {
+    this.tasks.splice(index - 1, 1);
   }
-};
+}
 
-renderOldTasks();
\ No newline at end of file
+const allTasks = new Tasks();
+allTasks.renderTasks();
+
+inputTask.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    const newTask = new Task(allTasks.tasks.length + 1, inputTask.value);
+    allTasks.addTask(newTask);
+    localStorage.setItem('tasks', JSON.stringify(allTasks.tasks));
+    allTasks.renderTask(newTask);
+    inputTask.value = '';
+  }
+});
+
+tasksList.addEventListener('click', (e) => {
+  if (e.target.classList.contains('edit-task-img')) {
```

Comment:
- [ ] Kindly fix the functionality of updating a task description. It's not working. When refreshing the screen the old data from local storage is loaded.

Reply:
Good job following the advice of the previous reviewer